src/Model_Hydro/CPU_Hydro/CPU_Shared_DataReconstruction.cpp

```diff
@@ -654,8 +745,11 @@ void Hydro_DataReconstruction( const real g_ConVar   [][ CUBE(FLU_NXT) ],
 
 
 #     if ( FLU_SCHEME == MHM )
+      real Extra_Flux[6][NCOMP_TOTAL_PLUS_MAG] = {0.0};
+      for (int f=0; f<6; f++)
```

Comment:
@ChunYen-Chen this looks like some unnecessary lines of code. Can it be deleted or were we supposed to use them?

Reply:
Yes, you can remove `Extra_Flux`.